Add general introduction for national pages (#147)
elixir-europe/infectious-diseases-toolkit: national-resources/index.md

title: National resources
---

> The national resources pages below provide a general overview of resources, tools, and data related to the management of infectious disease data in a given country. Given the recent COVID-19 pandemic, many of the resources featured on the pages are centered around COVID-19. However, the practices, regulations, and other resources can be repurposed for use with other pathogens.

Reviewer: "Given the recent..." is this going to age? Maybe delete "Given the recent COVID-19 pandemic,"?

Author: Suggested change to account for this

> The national resources pages below provide a general overview of resources, tools, and data related to the management of infectious disease data in a given country. Given the recent COVID-19 pandemic, many of the resources featured on the pages are centered around COVID-19. However, the practices, regulations, and other resources can be repurposed for use with other pathogens.

Author: ```suggestion
The national resources pages below provide a general overview of resources, tools, and data related to the management of infectious disease data in a given country. Many of the resources on the pages were developed during the COVID-19 pandemic and, as such, are focused on COVID-19. However, the practices, guidance, and other resources shown can be repurposed for use with other pathogens.
```

title: National resources
---

> The national resources pages provide a general overview of resources, tools, and information related to the management of infectious disease data in a given country. The aim of these pages is ultimately promote greater information exchange between countries so that they can benefit from each other's experiences. In particular, we hope that the descriptions of initiatives and resources developed in one country can provide valuable insights for those in other countries looking to create similar solutions for their own needs.

Reviewer 2: ```suggestion
The national resources pages provide a general overview of resources, tools, and information related to the management of infectious disease data in a given country. The aim of these pages is ultimately to promote greater information exchange between countries so that all can benefit from each other's experiences. In particular, we hope that the descriptions of initiatives and resources developed in one country can provide valuable insights for others looking to create similar solutions for their own needs.
```
